networkx/addons/__init__.py

```diff
@@ -0,0 +1 @@
+__import__('pkg_resources').declare_namespace(__name__)
```

Comment:
It felt that it can be empty, but now upon installation I realized it can't. Actually `networkx` and `networkx-metis` both share the common namespace `networkx.addons`. So, *both* of them need to carry this information.